lib/barbeque/message/job_execution.rb

```diff
@@ -13,7 +13,11 @@ def assign_body(message_body)
         super
         @application = message_body['Application']
         @job  = message_body['Job']
-        @body = message_body['Message']
+        @body = if message_body['Message'].is_a?(String)
```

Comment:
Then, let's remove it. Unnecessary code just increases maintenance cost. Future maintainers will be confused about its background and not be able to remove it easily for consistency once such a behavior is introduced.